.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
I am leaving it up to you to set the `interval` that works best for you. 

Options include: daily, weekly, monthly

I think that `weekly` is a good default but we can adjust as needed